README.md

```diff
@@ -104,3 +104,5 @@ Credits for Images:
 * Blueberry: Made by TheJYKoder (original)
 * BlueberryBushFull: Made by TheJYKoder(original)
 * Strawberry: Created by Patrick Wang. Original.
+* PineappleBushFull: Created MDJ117
```

Comment:
You might want to mention each of the three images you added. Also: you added the "BushFull" to two different places? It probably should only be under "blockTiles"

May also want to rename away from "Bush" - a pineapple doesn't really grow on a bush :D You could just name the stages something like "PineappleSapling", "ImmaturePineapple" and "MaturePineapple"

Finally you're missing an entry for the seed image and it needs to be waaaay shrunk down :-)